pkg/common/common.go

```diff
@@ -56,6 +56,11 @@ func ReplaceManifestUnSupportedAPIs(origManifest, mapFile string, kubeConfig Kub
 		return "", errors.Wrapf(err, "Failed to load mapping file: %s", mapFile)
 	}
 
+	// Append extra mappings to the config
+	for _, m := range extra {
+		mapMetadata.Mappings = append(mapMetadata.Mappings, m)
+	}
```

Comment:
```suggestion
	mapMetadata.Mappings = append(mapMetadata.Mappings, additionalMappings...)
```

Reply:
No need to iterate to append. Also recommend the more specific naming `additionalMappings`, rather than `extra`.

Reply:
🤦🏻 thanks!